templates/includes/components/sidebar_nav.html

```diff
@@ -1,4 +1,4 @@
-<ul class="p-sidebar-nav__list {% if not is_root %}js-mobile-no-collapse{% endif %}">
+<ul id="navigation" class="p-sidebar-nav__list u-clearfix {% if is_root %}js-mobile-no-collapse{% endif %}">
```

Comment:
Did you mean to inverse the `if is_root` logic? I thought we just reverted it back to `if not`